sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -443,7 +444,8 @@ case class Range(
     end: Long,
     step: Long,
     numSlices: Option[Int],
-    output: Seq[Attribute])
+    output: Seq[Attribute],
+    override val isStreaming: Boolean)
```

Comment:
how can a `Range` have data from a streaming source?

Reply:
I don't think there's necessarily a reason it shouldn't be able to; streaming sources are free to define getBatch() however they'd like.

Right now the only source actually doing that is a fake source in StreamSuite.